include/mlir/Analysis/Dominance.h

```diff
@@ -90,6 +90,9 @@ class DominanceInfo : public detail::DominanceInfoBase</*IsPostDom=*/false> {
   bool properlyDominates(Block *a, Block *b) {
     return super::properlyDominates(a, b);
   }
+
+  llvm::DomTreeNodeBase<Block> *getNode(Block *a);
+  void updateDFSNumbers();
```

Comment:
These are public APIs, can you provide documentation for them?

Reply:
Will do.